src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -73,6 +73,10 @@
     </Reference>
     <Reference Include="PresentationCore" />
     <Reference Include="PresentationFramework" />
+    <Reference Include="ProtoGeometry, Version=2.12.0.5056, Culture=neutral, processorArchitecture=AMD64">
+      <HintPath>..\packages\DynamoVisualProgramming.LibG_227_0_0.2.12.0.5056\lib\net48\ProtoGeometry.dll</HintPath>
```

Comment:
see below, there is already a reference to libG... where is this coming from?

Reply:
I fixed it here: https://github.com/DynamoDS/Dynamo/pull/11785

Reply:
@sm6srw you need to clean up your packages.config file for DynamoCoreWpf project. Unfortunately, this problem keeps happening as that file hasn't been checked in for some reason.

Reply:
Ok, I don't have a package.config file at all for DynamoCoreWpf and I can't find the file in git. I also noticed that package.config is added to .gitignore local to the project. Is this the way is should be? It is apparently not working.